### DISCUSSION TITLE
Disable hit-enter message on delete/create

### DISCUSSION THREAD

**Discussion by pseudoparenchymatous** [nvim-tree/nvim-tree.lua#1806] [Q&A] [answered]
How can I disable this?
![IMG_20221204_142529](https://user-images.githubusercontent.com/105337186/205478088-202aa120-0f46-427e-a6f9-c13ea75812f6.jpg)
![IMG_20221204_142601](https://user-images.githubusercontent.com/105337186/205478090-34c154ac-9fd4-4dd4-86a0-79a270916875.jpg)

**Comment by pseudoparenchymatous** [comment, marked as answer]
Found the option at
```
require("nvim-tree").setup({
    notify = {
        threshold = vim.log.levels.OFF
    }
})
```

**Reply by alex-courtis** [reply]
You could set it to ``WARNING` or `ERROR` so that you don't miss any important messages.
